README.md

```diff
@@ -13,19 +13,18 @@ Project Quay builds, stores, and distributes your container images.
 
 High-level features include:
 
-- Docker Registry Protocol [v1], [v2]
+- Docker Registry Protocol [v1 and v2]
```

Comment:
Since there apparently isn't a V1 page on Docker's site anymore, can you remove linking v1 to it?

Reply:
Sure thing. Just pushed something up.